app/src/main/java/com/rob729/quiethours/Activity/SplashScreen.kt

```diff
@@ -12,8 +12,7 @@ class SplashScreen : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        val intent = Intent(this, MainActivity::class.java)
-        startActivity(intent)
+        checkAppIntro()
```

Comment:
move the checkFirstRun() above checkAppIntro()

Reply:
Ok